codegenerator/cli/templates/static/codegen/src/EventModLookup.res

```diff
@@ -5,79 +5,86 @@ type errorKind = WildcardSighashCollision | Duplicate
 type eventError = {eventMod: eventMod, errorKind: errorKind}
 module ContractEventMods = {
   type t = {
+    mutable wildcard: option<eventMod>,
     all: array<eventMod>,
     byContractName: dict<eventMod>,
   }
 
   let empty = () => {
+    wildcard: None,
     all: [],
     byContractName: Js.Dict.empty(),
   }
 
   let isWildcard = (eventMod: eventMod) => {
     let module(Event) = eventMod
-    Event.handlerRegister->Types.HandlerTypes.Register.getEventOptions->(v => v.Types.HandlerTypes.wildcard)
+    Event.handlerRegister
+    ->Types.HandlerTypes.Register.getEventOptions
+    ->(v => v.Types.HandlerTypes.wildcard)
```

Comment:
This is the first time I see the syntax 😁

Reply:
Haha, I don't really have too strong an opinion but thought it flows better than using round brackets around the whole statement and accessing .wildcard at the end. I might use it again.